custom_components/sat/climate.py

```
@@ -575,7 +560,10 @@ def pulse_width_modulation_enabled(self) -> bool:
 
     @property
     def relative_modulation_value(self) -> int:
-        return self._maximum_relative_modulation if self._relative_modulation.enabled else MINIMUM_RELATIVE_MOD
+        if not self._relative_modulation.enabled and self._coordinator.supports_relative_modulation_management:
+            return MINIMUM_RELATIVE_MODULATION
```

Ruff (0.8.2)

`MINIMUM_RELATIVE_MODULATION` may be undefined, or defined from star imports

(F405)